web/src/main/kotlin/no/nav/su/se/bakover/web/routes/søknadsbehandling/LeggTilGrunnlagBosituasjonRoutes.kt

```diff
@@ -137,3 +148,43 @@ internal fun Route.leggTilGrunnlagBosituasjonRoutes(
         }
     }
 }
+
+internal fun SøknadsbehandlingService.KunneIkkeVilkårsvurdere.tilResultat(): Resultat {
+    return when (this) {
+        SøknadsbehandlingService.KunneIkkeVilkårsvurdere.FantIkkeBehandling -> fantIkkeBehandling
+        is SøknadsbehandlingService.KunneIkkeVilkårsvurdere.FeilVedValideringAvBehandlingsinformasjon -> this.feil.tilResultat()
+        SøknadsbehandlingService.KunneIkkeVilkårsvurdere.HarIkkeEktefelle -> harIkkeEktefelle
+    }
+}
+
+internal fun Søknadsbehandling.KunneIkkeOppdatereBosituasjon.tilResultat(): Resultat {
+    return when (this) {
+        is Søknadsbehandling.KunneIkkeOppdatereBosituasjon.KlarteIkkeHentePersonIPdl -> Feilresponser.fantIkkePerson
+        is Søknadsbehandling.KunneIkkeOppdatereBosituasjon.KunneIkkeLageGrunnlagsdata -> this.feil.tilResultat()
+        is Søknadsbehandling.KunneIkkeOppdatereBosituasjon.UgyldigTilstand -> ugyldigTilstand(this.fra, this.til)
+    }
+}
+
+internal fun KunneIkkeLageGrunnlagsdata.tilResultat(): Resultat {
+    return when (this) {
+        KunneIkkeLageGrunnlagsdata.FradragForEpsSomIkkeHarEPS -> HttpStatusCode.BadRequest.errorJson(
+            "Det er fradrag for EPS, når søker ikke har EPS",
```

Comment:
```suggestion
            "Kan ikke legge til fradrag knyttet til EPS for en bruker som ikke har EPS.",
```
Eller noe i den duren